components/create_comment/create_comment.jsx

```diff
@@ -465,17 +490,45 @@ class CreateComment extends React.PureComponent {
             this.setState({draft: updatedDraft});
         }
 
-        if (enableConfirmNotificationsToChannel && useChannelMentions &&
+        const notificationsToChannel = enableConfirmNotificationsToChannel && useChannelMentions;
+        let memberNotifyCount = 0;
+        let channelTimezoneCount = 0;
+        let mentions = [];
+        const notContainsAtChannel = !containsAtChannel(draft.message);
+        if (enableConfirmNotificationsToChannel && notContainsAtChannel) {
```

Comment:
i think a `useGroupMentions` check here would make sense?

Reply:
Mind explaining how I would go about doing this? Little confused

Reply:
After your explanation, I better understood and implemented. Please let me know what you think!